### PR TITLE
Add nitish and aniruddha to team syself

### PR DESCRIPTION
**Pull request by jschoone**

### CODE REVIEW

File: orgs/SovereignCloudStack/data.yaml [hunk 1]
```diff
@@ -57,6 +57,12 @@ members:
   - name: ""
     login: janiskemper
     role: member
+  - name: ""
```
Comment (garloff) [inline orgs/SovereignCloudStack/data.yaml:60]:
```suggestion
  - name:  Aniruddha Basak
```
Reasoning: The name is public on his github profile, so no need to hide it here.

File: orgs/SovereignCloudStack/data.yaml [hunk 2]
```diff
@@ -57,6 +57,12 @@ members:
...
+    login: aniruddha2000
+    role: member
+  - name: ""
```
Comment (garloff) [inline orgs/SovereignCloudStack/data.yaml:63]:
```suggestion
    - name:  Nitish Tiwari
```
dito

**Review by garloff — APPROVED** [review]
/lgtm though I prefer human being with names.

**Review by garloff — APPROVED** [review]
Sorry for my wrong spacing and thanks for fixing it up!

**Comment by jschoone** [comment]
> Sorry for my wrong spacing and thanks for fixing it up!

No worries

**Pull request merged**